services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from typing import List, Optional
+from django.db.models import DateTimeField
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: DateTimeField,
```

Comment:
The parameter `movie_show_time` should be of type `datetime`, not `DateTimeField`. `DateTimeField` is a Django model field type, whereas `datetime` is the appropriate type for a date-time value.